maestro-domain/src/main/java/bio/overture/maestro/domain/api/AnalysisCentricDocumentConverter.java

```diff
@@ -80,19 +80,57 @@ private static AnalysisCentricDonor mergeDonorBySpecimen(@NonNull List<AnalysisC
     checkNotFound(anyDonor.getSpecimens() != null && anyDonor.getSpecimens().size() > 0,
             "Failed to merge AnalysisCentricDonor by specimen: donor doesn't have specimen,");
 
+    //  Each donor has only one specimen in the list
```

Comment:
how was this determined ?

Reply:
each song Sample has only one specimen and one donor,
so after groupingby by donor id , there is only one specimen